### DISCUSSION TITLE
Continue project or use other free Event Platforms?

### DISCUSSION THREAD

**Discussion by Hanneses** [Hanneses/chapter2#30] [General]
I just found and forked this project from freeCodeCamp, did some work and some days later... I'm not quite sure if I continue this project. 

### Reason
1) The big event platform www.eventbrite.com just announced to provide their service without any charges if your event has no ticket costs/is free. This seems to be a really nice offer, since this is a professional platform with a lot of features. Furthermore there are other free to use event platforms like MeetUp.

2) Self-hosting an event platform (server, database, mail etc.) takes _at least_ 10 to 15$ per month (as far as I know) as well as ongoing effort for technical maintenance. This might be to much costs for small non-profit organizations.

### Further steps
I talk to some people working in non-profit organizations and ask for their needs. I hope to get a better understanding if a self-hosting platform is what they need or not. 

### What do you think?
What are your thoughts about that?
I'm looking forward to hear your opinions.

Worst case: chapter project is obsolete and we can use our energy for other nice projects. ;)

Best regards 
Hannes :)

**Comment by Thamira2001** [comment]
Hey hannesses I am Thamira, your absoloutely right eventbrite has  already kind of taken over event domain, maybe it is in our best interest to divert our efforts to something more useful, however if you still want to continue this project I am still willing to contribute, however if you want to pivot and need new ideas I have some amazing ideas that could really hit the nail in the market !

**Reply by Hanneses** [reply]
Okay, thanks. :)
Let's talk personally.

**Reply by Hanneses** [reply]
@Thamira2001, did you get my contact details?

**Comment by Hanneses** [comment]
Conclusion: I archive the project. Currently I see no need for such an self-hosted event platform. See **reasons** in discussion above.
